tests/relational_tests.rs

```diff
@@ -690,8 +965,6 @@ pub async fn linked() -> Result<(), DbErr> {
         )
         .group_by(baker::Column::Id)
         .group_by(Expr::col((Alias::new("r4"), customer::Column::Id)))
-        .group_by(baker::Column::Name)
-        .group_by(Expr::col((Alias::new("r4"), customer::Column::Name)))
```

Comment:
Not sure why you are removing these, but we should add new test cases, instead of changing existing ones.